webrtc/go/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
You probably do not want to commit the `.idea/` dir and its files, no?

Reply:
No probably not. Removing.